services/mailer/mail_test.go

```diff
@@ -96,6 +104,12 @@ func TestComposeIssueCommentMessage(t *testing.T) {
 	assert.Equal(t, "<user2/repo1/issues/1/comment/2@localhost>", gomailMsg.GetHeader("Message-ID")[0], "Message-ID header doesn't match")
 	assert.Equal(t, "<mailto:"+replyTo+">", gomailMsg.GetHeader("List-Post")[0])
 	assert.Len(t, gomailMsg.GetHeader("List-Unsubscribe"), 2) // url + mailto
+
+	var buf strings.Builder
+	gomailMsg.WriteTo(&buf)
+	mailMsg := strings.ReplaceAll(strings.ReplaceAll(buf.String(), "=", ""), "\r\n", "") // naïve quoted printable decoding, to allow string searching
```

Comment:
nit: use ascii "naive"

Reply:
Either way is fine, but https://english.stackexchange.com/a/12748/418296 calls it "if not actually archaic" 😆.

Reply:
They are talking about US context. When I studied the `naïve` was everywhere. 😏

Reply:
Which language is that? French or German?

Reply:
Let's keep it, golang files support unicode.

Reply:
Noticed I can just pass the whole mail source into the `quotedprintable.Reader` and not just the content part.